components/channel_header_dropdown/menu_items/toggle_favorite_channel/toggle_favorite_channel.test.tsx

```diff
@@ -12,6 +13,21 @@ describe('components/ChannelHeaderDropdown/MenuItem.ToggleFavoriteChannel', () =
     const baseProps = {
         channel: {
             id: 'channel_id',
+            display_name: 'channel_display_name',
```

Comment:
For future reference, we also accept if you just force TypeScript to treat this whole object as a Channel like
```typescript
const baseProps = {
    channel: {
        id: 'channel_id',
    } as Channel,
};
```
which could help save some typing if you're wanting to help with this further.